labelbox/schema/model.py

```diff
@@ -18,23 +18,26 @@ class Model(DbObject):
     name = Field.String("name")
     model_runs = Relationship.ToMany("ModelRun", False)
 
-    def create_model_run(self, name) -> "ModelRun":
+    def create_model_run(self, name, config=None) -> "ModelRun":
         """ Creates a model run belonging to this model.
 
         Args:
             name (string): The name for the model run.
+            config (json): Model run's training metadata config
         Returns:
             ModelRun, the created model run.
         """
         name_param = "name"
+        config_param = "config"
         model_id_param = "modelId"
         ModelRun = Entity.ModelRun
-        query_str = """mutation CreateModelRunPyApi($%s: String!, $%s: ID!) {
-            createModelRun(data: {name: $%s, modelId: $%s}) {%s}}""" % (
-            name_param, model_id_param, name_param, model_id_param,
-            query.results_query_part(ModelRun))
+        query_str = """mutation CreateModelRunPyApi($%s: String!, $%s: Json, $%s: ID!) {
+            createModelRun(data: {name: $%s, trainingMetadata: $%s, modelId: $%s}) {%s}}""" % (
+            name_param, config_param, model_id_param, name_param, config_param,
+            model_id_param, query.results_query_part(ModelRun))
         res = self.client.execute(query_str, {
             name_param: name,
+            config_param: config,
```

Comment:
Just curious, does this work okay when the param is None?

Reply:
Hm, let me test that

Reply:
While you're at it, could you add a test case for `config` being empty as well?

Reply:
yeah, thats what I meant! haha

Reply:
All good